security/vault.go

```diff
@@ -241,6 +242,10 @@ func (v *Vault) decrypt(data []byte) ([]byte, error) {
 	if err != nil {
 		return []byte{}, err
 	}
+
+	if !bytes.Contains(unpadMsg, []byte("!CHECK_ME!")) {
```

Comment:
Same here. A bit magic. Maybe move that to a separate constant with a tiny description? 😄